google/datalab/contrib/mltoolbox/commands/_ml.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Platform library - ml cell magic."""
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+try:
+  import IPython
+  import IPython.core.display
+  import IPython.core.magic
+except ImportError:
+  raise Exception('This module can only be loaded in ipython.')
+
+import pandas as pd
+import numpy as np
+import six
+
+import google.datalab.utils.commands
+import google.datalab.contrib.mltoolbox._local_predict as _local_predict
+
+
+@IPython.core.magic.register_line_cell_magic
+def ml(line, cell=None):
+  """Implements the datalab cell magic for MLToolbox operations.
+
+  Args:
+    line: the contents of the ml command line.
+  Returns:
+    The results of executing the cell.
+  """
+  parser = google.datalab.utils.commands.CommandParser(
+      prog='%ml',
+      description="""
+Execute MLToolbox operations.
+
+Use "%ml <command> -h" for help on a specific command.
+""")
+
+  predict_parser = parser.subcommand(
+      'predict', help='Predict with local or deployed models. ' +
+      'Prediction data is provided in cell in yaml format of CSV lines. For example, ' +
```

Comment:
The example only shows how to provide data in input cell, not by --data so should not be in "--data" section. I changed the help string to make it clearer.